src/System.IO.Pipelines/src/System/IO/Pipelines/ThreadPoolScheduler.netstandard.cs

```diff
@@ -9,11 +9,11 @@ namespace System.IO.Pipelines
 {
     internal sealed class ThreadPoolScheduler : PipeScheduler
     {
-        public override void Schedule<TState>(Action<TState> action, TState state)
+        public override void Schedule(Action<object> action, object state)
         {
             System.Threading.ThreadPool.QueueUserWorkItem(s =>
             {
-                var tuple = (Tuple<Action<TState>, TState>)s;
+                var tuple = (Tuple<Action<object>, object>)s;
```

Comment:
ValueTuple?

Reply:
For what benefit? It'll get boxed. I should just change this to Task.Factory.StartNew...

Reply:
Ah, netstandard2.0...

Don't change it to Task.Factory though; that brings in all sorts of extra mechanics